addons/website_theme_install/models/theme_models.py

```diff
@@ -107,10 +107,10 @@ def _convert_to_base_model(self, website, **kwargs):
         new_menu = {
             'name': self.name,
             'url': self.url,
-            'page_id': page_id,
+            'page_id': page_id and page_id[0].id or False,
             'new_window': self.new_window,
             'sequence': self.sequence,
-            'parent_id': parent_id,
+            'parent_id': parent_id and parent_id[0].id or False,
```

Comment:
Hello, your fix seems fine thanks but could you remove the `[0]` part which is useless? Thank you!

Also, instead of "wrong value for new_menu" in the commit message, could you use "use correct values for new menu creation" as the sentence must start with a verb for our guidelines.